sdk/servicebus/azure-servicebus/azure/servicebus/_common/auto_lock_renewer.py

```diff
@@ -109,16 +120,18 @@ def _auto_lock_renew(self, renewable, starttime, timeout, on_lock_renew_failure=
                 on_lock_renew_failure(renewable, error)
 
     def register(self, renewable, timeout=300, on_lock_renew_failure=None):
+        # type: (Union[ReceivedMessage, ServiceBusSession], float, Optional[LockRenewFailureCallback]) -> None
```

Comment:
dumb question (my lack of python knowledge):

Why LockRenewFailureCallback is defined as `LockRenewFailureCallback = Callable[[Union[ServiceBusSession, ReceivedMessage],Optional[Exception]], None]`, as we only take two parameters -- what does the last `None` represent?

Reply:
Return type :)